recipes/compas_view2/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "compas_view2" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ca50d7db4b4b049fd4472125c7d23bf4d6cff09fc74d63f02748fa988f64d324
+
+build:
+  number: 0
+  skip: true  # [py<36 or py>38]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  build:
+    - {{ cdt('mesa-libGL-devel') }}  # [linux]
```

Comment:
```suggestion
```

Reply:
but if i remove this the linux build fails because it can't find OpenGL...

Reply:
I think you added the yum requirements after the CDT was added? The CDT should only be needed if you're linking against OpenGL.